src/spanningTree/kruskal.cpp

```
@@ -0,0 +1,65 @@
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+ ********************************************************************PGR-GNU*/
+
+#include "spanningTree/kruskal.hpp"
+
+#include <vector>
+#include <set>
+
+namespace pgrouting {
```

Cppcheck (2.19.0)

[information] Include file

(missingIncludeSystem)